src/main/java/com/airsquared/blobsaver/model/Version.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019  airsquared
+ *
+ * This file is part of blobsaver.
+ *
+ * blobsaver is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * blobsaver is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with blobsaver.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.airsquared.blobsaver.model;
+
+public class Version implements Comparable<Version> {
```

Comment:
What's the need for adding a separate `Version` class? It seems unnecessary because all the version is every used for is checking if the version is the latest or not.

Reply:
What if I'm working on a beta version that's > latest published version? It doesn't hurt to have it and I already put in the effort to write it ;-; plus it might be useful if we add beta versions in the future cuz we could have  a class `BetaVersion extends Version` and make minimal changes to the logic to make betas work.